### PR TITLE
Import key modules at top level of hca package

### PR DESCRIPTION
**Pull request by kislyuk**
Also clean up hca namespace a bit.

### CODE REVIEW

File: hca/__init__.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 import os, logging
 
-import tweak
+from tweak import Config as _Config
```
Comment (ttung) [inline hca/__init__.py]: why this change?

File: hca/__init__.py [hunk 2]
```diff
@@ -24,3 +24,5 @@ def get_config():
 
 
 logger = logging.getLogger(__name__)
+
+from . import dss, upload  # noqa
```
Comment (ttung) [inline hca/__init__.py]: Rather than do the `# noqa` thing, you can move the code in `__init__.py` into a module in the hca package.

File: hca/__init__.py (hunk 1 above)
Comment (kislyuk) [reply hca/__init__.py]: I'm importing it with an underscore because the Config object should not be part of this package's API.

File: hca/__init__.py (hunk 2 above)
Comment (kislyuk) [reply hca/__init__.py]: Good point, I'll try and refactor this.

File: hca/__init__.py (hunk 1 above)
Comment (ttung) [reply hca/__init__.py]: Ah!  Makes sense.
